src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,59 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final String SEPARATOR = ",";
+    private static final int FIRST = 0;
+    private static final int SECOND = 1;
```

Comment:
Consider giving meaningful names to your constants.
Naming constants just by their value is a missed opportunity to make the code more self-explanatory.
The name of a constant should indicate its role or purpose within the application, providing context to other developers.

Instead of:
// private static final int ZERO = 0;
// private static final int FIVE = 5;
// private static final String DASH = "-";

Consider something like:
// private static final int AMOUNT_INDEX = 0;
// private static final int DEFAULT_VALUE = 5;
// private static final Strint DELIMITER = "-";